Source/Utils/WarpXConst.H

```diff
@@ -39,6 +39,7 @@ namespace PhysConst
     static constexpr auto m_u   = static_cast<amrex::Real>( 1.66053906660e-27 );
     static constexpr auto hbar  = static_cast<amrex::Real>( 1.054571817e-34 );
     static constexpr auto alpha = static_cast<amrex::Real>( 0.007297352573748943 );//mu0/(4*MathConst::pi)*q_e*q_e*c/hbar;
+    static constexpr auto a0    = static_cast<amrex::Real>( 5.29177210903e-11 );//hbar/(m_e*c*alpha)
```

Comment:
This, `a0`, doesn't seem to be used anywhere.

Reply:
True. I used it before when I was looking at the Wentzel model but decided to skip that implementation. I'll remove the `a0` addition.